### PR TITLE
post: v23.4.0 announcement

### PR DESCRIPTION
**Pull request by aduh95**
Refs: https://github.com/nodejs/node/pull/56119

### CODE REVIEW

File: records/new/v23.4.0-announcement.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "action": "post",
+  "account": "NODEJS_ORG",
+  "richText": "Thanks to @targos.dev and @aduh95.bsky.social for a new Node.js Current release: [v23.4.0](https://nodejs.org/en/blog/release/v23.4.0)"
```
Comment (joyeecheung) [inline records/new/v23.4.0-announcement.json:4]: Hmm, actually I don’t think the markdown syntax works with rich text detection. It should just be
```suggestion
  "richText": "Thanks to @targos.dev and @aduh95.bsky.social for a new Node.js Current release: https://nodejs.org/en/blog/release/v23.4.0"
```

**Comment by joyeecheung** [comment]
I am typing on my phone so I don't know if these fit in 300 chars, but I think this is the list that can be put into the post (maybe we can cut some if it's too long, or we can send a reply to the first post)

- assert.partialDeepStrictEqual()
- --trace-env
- net.BlockList.isBlockList()
- net.SocketAddress.parse()
- BlockList support in net.Server and net.connect
- unflag node:sqlite
- StatementSync.prototype.iterate()
- deprecate features.{ipv6,uv} and features.tls_*
- new collaborator @jordan.har.band

**Comment by aduh95** [comment]
Yeah the following text is already 283 char, it would need to be a response – which I think needs to be another PR, I don't think we support making threads

```markdown
Notable changes:

- `assert.partialDeepStrictEqual()`
- `--trace-env`
- `net.BlockList.isBlockList()`
- `net.SocketAddress.parse()`
- BlockList support in `net.Server` and `net.connect`
- unflag `node:sqlite`
- `StatementSync.prototype.iterate()`
- new Collaborator @jordan.har.band
```

**Comment by joyeecheung** [comment]
How about

```
Thanks to @targos.dev and @aduh95.bsky.social for a new Node.js Current release!

https://nodejs.org/en/blog/release/v23.4.0

- assert.partialDeepStrictEqual()
- --trace-env
- net.SocketAddress.parse()
- Better net.BlockList support
- unflag node:sqlite
- StatementSync.prototype.iterate()
```

It's 289 chars, and we can add `new Collaborator @jordan.har.band` to a separate post

**Comment by joyeecheung** [comment]
(Also another note, we can support file reference in the JSON, so that the rich text content can be put into a .txt, and the workflow can read it and JSON.stringify it)

File: records/new/v23.4.0-announcement.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "action": "post",
+  "account": "NODEJS_ORG",
+  "richText": "Thanks to @targos.dev and @aduh95.bsky.social for a new Node.js Current release: https://nodejs.org/en/blog/release/v23.4.0"
```
Comment (aduh95) [inline records/new/v23.4.0-announcement.json:4]:
```suggestion
  "richText": "Thanks to @targos.dev and @aduh95.bsky.social for a new Node.js Current release!\n\nhttps://nodejs.org/en/blog/release/v23.4.0\n\n- assert.partialDeepStrictEqual()\n- `--trace-env`\n- net.SocketAddress.parse()\n- Better net.BlockList support\n- Unflag `node:sqlite`\n- StatementSync.prototype.iterate()\n"
```

**Comment by joyeecheung** [comment]
I tested it in https://github.com/joyeecheung/bluesky-playground/pull/11 which triggered https://bsky.app/profile/pixel-voyager.bsky.social/post/3lcxuadloj62p - looks good!

**Comment by joyeecheung** [comment]
It seems the branch deletion is automatic, after some googling I found

https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-the-automatic-deletion-of-branches

So I disabled it in the repo setting.

**Comment by joyeecheung** [comment]
About `new Collaborator @jordan.har.band`

It seems a repost of https://bsky.app/profile/jordan.har.band/post/3lcixvbt24k2e is more apt?

**Pull request merged**